Notes.txt

```diff
@@ -11,4 +11,11 @@ Notes for this year:
 
```

Comment:
I'd recommend moving notes to a more organized 'docs' folder, organized notes by type (example: "pneumatics", "sub-systems", things like that)